sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2013,6 +2014,19 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("SPARK-34157 Unify output of SHOW TABLES and pass output attributes properly") {
+    withTable("tbl") {
+      sql("CREATE TABLE tbl(col1 int, col2 string) USING parquet ")
```

Comment:
@beliefer, do you mind showing the output before after this PR in the PR description? it would be much easier to review and follow what change you're proposing.

Reply:
OK